frontend/components/Menubar.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactElement, useEffect, useState } from 'react';
+import Sidebar from './Sidebar';
+import NavBar from './NavBar';
+import { size } from '../templates/mediaSizes';
+
+const Menubar = (): ReactElement => {
+  const windowSize = useWindowSize();
+
+  if (!windowSize.width) return <Sidebar />;
+
+  return windowSize.width < size.laptop ? <Sidebar /> : <NavBar />;
+};
+
+// Hook - https://usehooks.com/useWindowSize/
+function useWindowSize() {
```

Comment:
awesome idea. lets place this function inside a `hooks` directory though, just in case we want to reuse it